rules/okta/okta_attempt_to_modify_or_delete_application_sign_on_policy.toml

```diff
@@ -0,0 +1,35 @@
+[metadata]
+creation_date = "2020/07/01"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/01"
+
+[rule]
+author = ["Elastic"]
+description = """
+An adversary may attempt to modify or delete the sign on policy for an Okta application in order to remove or weaken an
+organization's security controls.
+"""
+false_positives = [
+    """
+    Consider adding exceptions to this rule to filter false positives if sign on policies for Okta applications are
+    regularly modified or deleted in your organization.
+    """,
+]
+index = ["filebeat-*"]
+language = "kuery"
+license = "Elastic License"
+name = "Attempt to Modify or Delete Okta Application Sign On Policy"
```

Comment:
Based off the Okta docs, it looks like it only logs on success?
We could cut some of the language down, since we don't need the "attempt" disclaimer.

```suggestion
name = "Modification or Removal of an Okta Application Sign On Policy"
```

Reply:
I'm fine with this change to the rule name, but the reason I left the `event.outcome` value out of some of these rules is that it's not clear what outcomes/results can be logged for each event.

<img width="853" alt="image" src="https://user-images.githubusercontent.com/56409778/86302380-be11a400-bbc5-11ea-924c-3a43d255ac9a.png">